README.md

```diff
@@ -11,30 +11,29 @@ Requirements
 
 Some Useful git Commands 
 --------------------
-git remote -v
-      Show all the tracked repositories
+`git remote -v` - Show all the tracked repositories
 
-git remote add upstream git@github.com:Code-Collaborate-Community/baboon.git
```

Comment:
GH org name should be changed

Reply:
@ipolonsk change that to:
git remote add upstream git@github.com:beyond-io/baboon.git

Reply:
Changed and commit